### PR TITLE
Initial setup of project

### PR DESCRIPTION
**Pull request by hydraadra112**
I created `.tsx` files for each page, and modified key files similar to our previous web app.

### CODE REVIEW

File: frontend/src/index.css
Comment (computerscience-person) [inline frontend/src/index.css:1]: We don't have tailwind yet, remove your changes here and amend your commit.

Comment (computerscience-person) [reply frontend/src/index.css:1]: Also, no need na ang other css stuff kay may ara naman css reset ang tailwind mismo.

Comment (hydraadra112) [reply frontend/src/index.css:1]: I assume the rest are okay?

File: frontend/src/App.tsx
Comment (computerscience-person) [inline frontend/src/App.tsx:1]: Just remove the other stuff nalang

**Review by computerscience-person — CHANGES_REQUESTED** [review]
Amend your previous commits with the changes suggested.

[Amending commits](https://www.atlassian.com/git/tutorials/rewriting-history)

File: frontend/src/index.css
Comment (computerscience-person) [reply frontend/src/index.css:1]: Can just remove everything tbh.

File: frontend/src/App.tsx
Comment (hydraadra112) [reply frontend/src/App.tsx:1]: Remove the waterup code or the highlighted code?

**Review by computerscience-person — APPROVED** [review]

**Pull request merged**